test/tools/router_check/validation.proto

```diff
@@ -65,7 +65,9 @@ message ValidationInput {
   // Additional headers to be added as input for route determination.
   // The “:authority”, “:path”, “:method”, “x-forwarded-proto”, and “x-envoy-internal” fields are
   // specified by the other config options and should not be set here.
-  repeated envoy.config.core.v3.HeaderValue additional_headers = 8;
+  repeated envoy.config.core.v3.HeaderValue additional_headers = 8 [deprecated = true];
```

Comment:
Unless we support the deprecated fields somehow, I don't think it actually makes sense to deprecate here as the user's config will load but it won't actually work correctly? I think I would just reserve the fields that are not used anymore assuming we don't have to do back compat?

Reply:
removed/reserved the fields